### PR TITLE
Change PayRate start/end from datetime to date.

### PR DESCRIPTION
**Pull request by lydiagu**

### CODE REVIEW

File: orchestra/models.py [hunk 1]
```diff
@@ -554,19 +555,47 @@ class PayRate(models.Model):
...
+
+    def save(self, *args, **kwargs):
+        if self.end_date and self.end_date < self.start_date:
+            raise ModelSaveError('end_date must be greater than '
+                                 'start_date')
+
+        if self.end_date is None:
+            # If end_date is None, need to check that no other PayRates have
+            # end_date is None, nor do they overlap.
+            if PayRate.objects.filter(
```
Comment (marcua) [inline orchestra/models.py]: do we have to exclude self.id if it exists?

File: orchestra/models.py [hunk 2]
```diff
@@ -554,19 +555,47 @@ class PayRate(models.Model):
         hourly_multiplier (decimal.Decimal):
             Any multiplier that gets applied to a worker's pay
             (e.g., fees imposed by a payment gateway).
-        start_datetime (datetime.datetime):
+        start_date (datetime.date):
```
Comment (marcua) [inline orchestra/models.py:558]: note that it's inclusive

File: orchestra/models.py [hunk 3]
```diff
@@ -554,19 +555,47 @@ class PayRate(models.Model):
...
             The beginning of the time period of the pay rate.
-        end_datetime (datetime.datetime):
+        end_date (datetime.date):
```
Comment (marcua) [inline orchestra/models.py:560]: note that it's inclusive

File: orchestra/migrations/0041_alter_payrate_fields.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        ('orchestra', '0040_timeentry_created_at'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='payrate',
+            name='end_datetime',
+            field=models.DateField(blank=True, null=True),
+        ),
+        migrations.AlterField(
```
Comment (marcua) [inline orchestra/migrations/0041_alter_payrate_fields.py:21]: does this truncate correctly?

File: orchestra/models.py (hunk 1 above)
Comment (lydiagu) [inline orchestra/models.py]: oo good point

File: orchestra/migrations/0041_alter_payrate_fields.py (hunk 4 above)
Comment (lydiagu) [inline orchestra/migrations/0041_alter_payrate_fields.py:21]: yes

**Pull request merged**